File: Exercícios de Lógica/Java/Extra/Exrcicios_Java/src/Exercicio13.java
/*
Exercicio 013
Tendo como dado de entrada a altura (h) de uma pessoa,
construa um algoritmo que calcule seu peso ideal, utilizando as seguintes fórmulas:
 Para homens: (72.7h) - 58
 Para mulheres: (62.1h) - 44.7
 */

import java.util.Scanner;

public class Exercicio13 {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        System.out.print("Digite sua altura: ");
        float h = scanner.nextFloat();

        float peso_ideal_homem = pesoIdealHomem(h);
        float peso_ideal_mulher = pesoIdealMulher(h);

        System.out.printf("\nSe você for homem o seu peso ideal é: %.2f Kg", peso_ideal_homem);
        System.out.printf("\nSe você for mulher o seu peso ideal é: %.2f Kg", peso_ideal_mulher);
        scanner.close();

    }

    public static float pesoIdealHomem(float altura){
    return (float) (72.7*altura - 58);
    }

    public static float pesoIdealMulher(float altura){
        return (float) (62.1*altura - 44.7);
    }
}

File: Exercícios de Lógica/Python/Exercicio13.py
'''
Exercicio 013

Tendo como dado de entrada a altura (h) de uma pessoa, construa um algoritmo que calcule seu peso ideal, utilizando as seguintes fórmulas: Para homens: (72.7h) - 58 Para mulheres: (62.1h) - 44.7
'''

h = float(input("Digite sua altura: "))
sexo=input("Digite o seu sexo (M/F): ").lower()

if sexo =="m":
    peso_ideal=(72.7*h)-58
    print(f"Seu peso ideal é {peso_ideal} Kg")

elif sexo == "f":
    peso_ideal=(62.1*h-44.7)
    print(f"Seu peso ideal é {peso_ideal} Kg")

else:
    print("Erro, por favor digite corretamente sua altura e seu sexo.")
